### ISSUE TITLE
no attribute 'id' Error

### ISSUE THREAD

**Issue by MarlonAbeykoon** [mathandpencil/django-bulk-sync#25]
I started using this library today and get the following error when trying to `bulk_sync`

```
{taskinstance.py:1150} ERROR - 'Clients' object has no attribute 'id'
Traceback (most recent call last):
  File "/Users/marlonabeykoon/.pyenv/versions/3.6.12/envs/airenv/lib/python3.6/site-packages/airflow/models/taskinstance.py", line 984, in _run_raw_task
    result = task_copy.execute(context=context)
  File "/Users/marlonabeykoon/Code/AirflowDags/operators/DjangoOperator.py", line 50, in execute
    skip_deletes=True
  File "/Users/marlonabeykoon/.pyenv/versions/3.6.12/envs/airenv/lib/python3.6/site-packages/bulk_sync/__init__.py", line 95, in bulk_sync
    new_obj.id = old_obj.id
```

Code

```
        data = [
            self.model(client_id=id, client_name=name)
            for id, name in raw_data
        ]
        ret = bulk_sync(
            new_models=data,
            filters=[],
            fields=['client_name'],
            key_fields=('client_id',),
            skip_deletes=True
        )
```
Model

```
class Clients(models.Model):
    client_id = models.PositiveIntegerField(primary_key=True)
    client_name = models.CharField(max_length=255, null=True)
```

I want to update/save the client_name. `client_id` is the primary key.
What have I done wrong here?

**Comment by ses4j** [comment]
Nothing.  I am releasing a patch to support `pk` that aren't `id`..

**Comment by ses4j** [comment]
Fixed with v3.2.0.

**Comment by MarlonAbeykoon** [comment]
Thanks, it works.
